pkg/cli/clisqlexec/run_query.go

```diff
@@ -42,7 +42,11 @@ func (sqlExecCtx *Context) RunQuery(
 // Timings is enabled are written to 'tw'.
 // Errors and warnings, if any, are printed to 'ew'.
 func (sqlExecCtx *Context) RunQueryAndFormatResults(
-	ctx context.Context, conn clisqlclient.Conn, w, tw, ew io.Writer, fn clisqlclient.QueryFn,
+	ctx context.Context,
+	conn clisqlclient.Conn,
+	w, tw, ew io.Writer,
+	fn clisqlclient.QueryFn,
+	escapeNewline bool,
```

Comment:
This can be a new field on the Context struct.

Reply:
`cli.sqlExecCtx` is used in many places so this field can't be moved there unless we either
1) make a new instance for use by zip.go
2) want to affect all places using it